index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Website</title>
+    <link rel="stylesheet" href="./stylesheet/style.css">
+</head>
+
+<body>
+
+    <nav>
+        <h4>David Kasilia Logo</h4>
+        <img class="menu" src="./images/menu_icon-removebg-preview.png" alt="Menu Icon">
+    </nav>
+
```

Comment:
- Please The section should have the margins (top, bottom, left, right) defined in the design

| Your Design | Figma design |
| ---------------- | ------------------ |
| ![image](https://user-images.githubusercontent.com/74744787/176552643-6bd65c81-ec91-4bb6-97e7-b6080368f486.png) | ![image](https://user-images.githubusercontent.com/74744787/176552745-19dce0aa-8f16-49f5-b00a-195bc5546a8c.png) |